src/main/scala/com/tapad/docker/ComposeFile.scala

```diff
@@ -247,6 +255,19 @@ trait ComposeFile extends SettingsHelper with ComposeCustomTagHelpers with Print
     }
   }
 
+  def getStaticPortMappings(ports: Seq[String]): java.util.ArrayList[String] = {
+    val staticPorts = ports.map { port =>
+      val portArray = port.split(":")
+      if (portArray.length == 2) {
```

Comment:
You should put the `0` into a variable called `dynamicPortIdentifier` or something like that.